test/hotspot/jtreg/ProblemList.txt

```diff
@@ -98,6 +98,7 @@ runtime/os/TestTracePageSizes.java#compiler-options 8267460 linux-aarch64
 runtime/os/TestTracePageSizes.java#G1 8267460 linux-aarch64
 runtime/os/TestTracePageSizes.java#Parallel 8267460 linux-aarch64
 runtime/os/TestTracePageSizes.java#Serial 8267460 linux-aarch64
+runtime/os/TestTrimNative.java 8312525 linux-all
```

Comment:
That entry won't work. It should be:
```
runtime/os/TestTrimNative.java#trimNative 8312525 linux-all
```

Reply:
The failure in Mach5 is reported as "runtime/os/TestTrimNative.java#trimNative"
so you have to include the sub-test portion. As far as I can see, that's the only
sub-test that is failing.

FYI: Simply naming the test, e.g., "runtime/os/TestTrimNative.java" does NOT
disable all of the sub-tests.

Reply:
Arrgh. This should be something we should allow, maybe with wildcards.

Thanks, Dan, I changed the test to include all tests by name that could be possibly affected by this bug.

Reply:
[CODETOOLS-7902712](https://bugs.openjdk.org/browse/CODETOOLS-7902712) is one the issues on this topic.